tests/Integration/RecommendedApiTest.php

```diff
@@ -10,6 +10,9 @@
 namespace Parsely\Tests\Integration;
 
 use Parsely\UI\Recommended_Widget;
+use Parsely\Recommended_Content;
+
+require_once __DIR__ . '/../../src/class-recommended-content.php';
```

Comment:
This require shouldn't be needed - all of the file-loading for running tests should either already be handled, or it should be handled in a bootstrap file.